src/main/java/com/spotify/reaper/resources/view/RepairRunStatus.java

```diff
@@ -288,4 +336,24 @@ public String getLastEvent() {
   public void setLastEvent(String lastEvent) {
     this.lastEvent = lastEvent;
   }
+
+  public String getDuration() {
+    return duration;
+  }
+
+  public void setDuration(String duration) {
+    this.duration = duration;
+  }
+
+  @JsonProperty("estimate_time_of_arrival")
```

Comment:
Duration of completed run. It only applies to DONE and ERROR repairs. What's a better name? running_time?